census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -315,6 +315,12 @@ public void streamCreated(Attributes transportAtts, Metadata headers) {
         headers.discardAll(tracingHeader);
         headers.put(tracingHeader, span.getContext());
       }
+      span.addAnnotation("Real stream created.");
+    }
+
+    @Override
+    public void createPendingStream() {
+      span.addAnnotation("Pending stream created.");
```

Comment:
I don't this annotation gains us anything, as this will be the same as the start of the span.

Reply:
For `The start of the span` you mean creating when the stream tracer is created by the census interceptor, right?
That is around the same time as pending stream is created. The difference is that every stream gets called on span created, but only pending stream gets called with `createPendingStream`.

Reply:
discussed offline, this needs to be added on parent span. Plus creatingPendingStream is not  sufficiently mean name resolution delay. Maybe we will add follow up PR about name resolution delay timestamp.